### PR TITLE
Added null value where no dialling code

### PR DESCRIPTION
**Pull request by joefhall**

### CODE REVIEW

File: src/config/countries.php [hunk 1]
```diff
@@ -933,7 +933,7 @@
     ],
     'UM' => [
         'name' => 'United States Outlying Islands',
-        'diallingCode' => ,
+        'diallingCode' => null,
```
Comment (mikefrancis) [inline src/config/countries.php]: Take it you've checked them all?

Comment (joefhall) [reply src/config/countries.php]: @mikefrancis Yep I have been through them otherwise

**Comment by joefhall** [comment]
@mikefrancis Could you re-review this please?  I made some changes to fix tests failing.  (I've also seen the CodeCov message above - speaking to Ben to try and work it out.)

File: src/Providers/CountriesServiceProvider.php [hunk 2]
```diff
@@ -22,7 +24,7 @@ class CountriesServiceProvider extends ServiceProvider
     public function register()
     {
         /** @var \Illuminate\Config\Repository $config */
-        $config = $this->app['config'];
+        $config = $this->app->config;
```
Comment (mikefrancis) [inline src/Providers/CountriesServiceProvider.php]: Are you 100% sure this is right? `$this->app['config']` will get the right binding, I'm not sure what the property `$config` is for...

File: tests/Providers/TestCountriesServiceProvider.php [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
     /**
-     * Set up mocks.
+     * Set up tests.
      *
      * @return void
      */
     public function setUp()
     {
-        $this->app = new ApplicationStub();
+        $this->appMock = Mockery::mock(Application::class);
```
Comment (mikefrancis) [inline tests/Providers/TestCountriesServiceProvider.php]: Could use: 
```php
$this->appMock = $this->createMock(Application::class);
```

Which should theoretically do the same and means you can remove `mockery/mockery` from the `require-dev`

File: src/Providers/CountriesServiceProvider.php (hunk 2 above)
Comment (joefhall) [reply src/Providers/CountriesServiceProvider.php]: Yep, I've checked and this looks to be just another way of referencing exactly the same thing.

File: tests/Providers/TestCountriesServiceProvider.php (hunk 3 above)
Comment (joefhall) [reply tests/Providers/TestCountriesServiceProvider.php]: Thanks, good shout - have switched to PHPUnit mocks now and removed the need for Mockery.

**Pull request merged**